tests/algorithm/lin_interp_test.cpp

```diff
@@ -292,7 +53,7 @@ TEST_CASE("lin_interp_api", "lin_interp")
   const int km1 = 0;
   const int km2 = 0;
 
-  using LIV = ekat::LinInterp<Real,EKAT_TEST_POSSIBLY_NO_PACK_SIZE>;
```

Comment:
This is another thing: I unilaterally decided to remove the concept of "small pack" and "possibly no pack". I feel like those are application-side choices, and they were making the cmake code a bit hard to read/follow.

We are simply testing that things work here, so I thought there was little to gain in keeping lots of cmake configurable pack choices...

Reply:
This is disruptive in the sense that EAMxx has to adapt to these changes, right? Not a big deal, especially if the same people are doing the work. :-)

Reply:
Well, not really. EAMxx is still free to have multiple pack sizes, and use different sizes in different places. We just don't test that in ekat.